<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ruleset_widget</class>
 <widget class="QWidget" name="ruleset_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Dealer Hits on Soft 17</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Dealer Peek</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>Push on Dealer 22</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="blackJackPayout"/>
    </item>
    <item>
     <widget class="QSpinBox" name="numDecks"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>Double After Split</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>Resplit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string>Hit Split Aces</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="text">
       <string>Resplit Aces</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="text">
       <string>Surrender</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
